README.md

```diff
@@ -139,3 +140,14 @@ let start = firstCell.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
 let finish = firstCell.coordinateWithNormalizedOffset(CGVectorMake(0, 6))
 start.pressForDuration(0, thenDragToCoordinate: finish)
 ````
+
+### Move cursor position within text field
+
+In this example, a table view cell contains a text field with some content. To move the cursor within the text field, create a `XCUICoordinate` from the left edge of the text field, tap & hold and then drag `dx` points to the right.
+
+````swift
+let textField = cells.textFields.elementBoundByIndex(1)
+let start = textField.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
+let finish = start.coordinateWithOffset(CGVectorMake(70, 0))
```

Comment:
you're right: the hard-coded 70 is only for illustration purpose - which might or might not work, depending on the text
I could add this to the paragraph, describing the context of the example - would that be clearer?
or, I could add a line of code to set the content of the textField - but I feel that would be distracting